src/app/data/loaders/iumLoader.ts

```diff
@@ -0,0 +1,25 @@
+import type { FileCollection } from 'filelist-utils';
+
+import {
+  getEmptyMeasurements,
+  Loader,
+  Measurements,
+  mergeMeasurements,
+} from '../DataState';
+
+import { assert } from '@/utils/assert';
+
+export const iumLoader: Loader = async function iumLoader(
```

Comment:
You can remove the ium loader entirely. An `ium` file should never be handled unless it's the only file that was dropped. You can think of an `ium` file in our application as of a `psd` file in photoshop, if the analogy helps.

Reply:
fixed